fieldpath/element.go

```diff
@@ -24,6 +24,58 @@ import (
 	"sigs.k8s.io/structured-merge-diff/value"
 )
 
+type KeyValue struct {
+	Key   string
+	Value value.Value
+}
+
+func SortKeyValues(kv []KeyValue) {
```

Comment:
Godoc comments on public things in this file?

Reply:
I'm curious if I should make a type for `[]KeyValue` and make that a method of the type.

Reply:
Alternative interface if you make a type: `func (kvl KeyValueList) Sort() { ... }`

Reply:
Yeah, that's what I had in mind